engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/command/DestroyContextCmd.scala

```diff
@@ -35,17 +35,18 @@ class DestroyContextCmd(
 
   private def removeContext()(implicit ctx: RuntimeContext): Unit = {
     ctx.jobControlPlane.abortJobs(request.contextId)
+    val contextLock = ctx.locking.getOrCreateContextLock(request.contextId)
```

Comment:
Why isn't the `contextLock` a field in `RuntimeContext`?

Reply:
> `RuntimeContext` is not per context id .... `RuntimeContext` is a case class, so immutable

I see. `RuntimeContext` holds `locking`, the `contextId` isn't part of `RuntimeContext`. In such case, there is no point of putting `contextLock` into `RuntimeContext`. I must have seriously confused myself. Sorry.